dom-manipulation/main.js

```diff
@@ -0,0 +1,27 @@
+var clickCount = 0;
+var $hotButton = document.querySelector('.hot-button');
+var $clickCount = document.querySelector('.click-count');
+
+function clickCounter() {
+  clickCount++;
+  $clickCount.textContent = 'Clicks: ' + clickCount;
+}
+
+function buttonTemp() {
+  if (clickCount < 4) {
+    $hotButton.className = 'hot-button cold';
+  } else if (clickCounter < 7) {
+    $hotButton.className = 'hot-button cool';
+  } else if (clickCount < 10) {
+    $hotButton.className = 'hot-button tepid';
+  } else if (clickCount < 13) {
+    $hotButton.className = 'hot-button warm';
+  } else if (clickCount < 16) {
+    $hotButton.className = 'hot-button hot';
+  } else {
+    $hotButton.className = 'hot-button nuclear';
+  }
+}
+
+$hotButton.addEventListener('click', clickCounter);
+$hotButton.addEventListener('click', buttonTemp);
```

Comment:
Instead of having two separate event handlers for the `click` event on this one element, you should do all of this work in one function, especially because the work is related to one another.